venter/tests.py

```diff
@@ -201,3 +214,49 @@ def test_model_venter(self):
         )
 
         self.assertEqual(str(complaintMedia), 'www.google.com')
+
+        authority = Authorities.objects.create(
+            name='dummyauth',
+            email='dummyauth@example.com'
+        )
+
+        self.assertEqual(str(authority), 'dummyauth@example.com')
+
+    def test_admin_actions(self):
+        self.complaint_admin = ComplaintModelAdmin(Complaints, AdminSite())  # pylint: disable=W0201
+
+        def create_complaint(user, **kwargs):
+            Complaints.objects.create(created_by=user, **kwargs)
+
+        create_complaint(self.user.profile, status='Reported')
```

Comment:
Turned these into simple Complaints.objects.create() to avoid issues

Reply:
I think you misunderstood me. I meant that you should define constants like
```
STATUS_REPORTED = 'Reported'
STATUS_IN_PROGRESS = 'In Progress'
```
etc. and then use these so,
`Complaints.objects.create(created_by=self.user.profile, status='Reported')`
would change to
`Complaints.objects.create(created_by=self.user.profile, status=STATUS_REPORTED)`
which is much less prone to error (since this will fail tests immediately if there is a spelling error).